app/views/layouts/_content.html.haml

```diff
@@ -15,7 +15,8 @@
         .row.toolbar-pf#toolbar
           .col-md-12
             .toolbar-pf-actions
-              = render :partial => "layouts/x_taskbar"
+              #taskbar_buttons_div-x
```

Comment:
Pretty sure you can get rid of `#taskbar_buttons_div-x`, it's never selected on..